tla/consensus/abs.tla

```diff
@@ -0,0 +1,58 @@
+---- MODULE abs ----
+\* Abstract specification for a distributed consensus algorithm.
+\* Assumes that any node can inspect the state of all other nodes. 
+
+EXTENDS Sequences, SequencesExt, Naturals, FiniteSets
+
+CONSTANT Servers, Terms, MaxLogLength, StartTerm
+
+\* Commit logs from each node
+\* Each log is append-only and the logs will never diverge.
+VARIABLE CLogs
```

Comment:
This human linter would have lower-cased the variable name.

Reply:
That does seem to be the de facto convention in our specs, e.g. https://github.com/microsoft/CCF/blob/main/tla/consensus/ccfraft.tla#L241, https://github.com/microsoft/CCF/blob/main/tla/consistency/Consistency.tla#L135

camelCase for VARIABLES, and CamelCase for CONSTANTS.

Reply:
I have no strong opinion on this but would prefer to keep all specs constant

Reply:
commitLogs would be consistent with other specs

Reply:
https://github.com/microsoft/CCF/pull/6438/commits/2ab3dfec0eb1a1be140e845624a60f9c5d992707